docs/UserGuide.md

```diff
@@ -72,11 +72,43 @@ Tinner aims to solve the hassle of managing internship applications though 3 asp
 --------------------------------------------------------------------------------------------------------------------
 # About <a id="about"></a>
 
-This section helps you to understand the Graphical User Interface (GUI), commonly used technical terminologies, general symbols and information about the command format. 
-
 ## Outline of this Guide <a id="outline"></a>
+The User Guide's primary goal is to assist the user in learning how to use the application Tinner
+
+This subsection helps you to understand the [Graphical User Interface (GUI)](#navigation), commonly used [technical terminologies](#technical-terminologies) and [general symbols](#general-symbols) to help you better understand the User Guide.
+
+Following this would be the [Features](#features), detailing the capabilities of Tinner and how to use them. This section can be either read from the start to bottom or by nicely divided segments of Tinner's repertoire in the [Table of Contents](#toc) and learn everything you can do with that specific area.
 
 ## Navigating within Tinner <a id="navigation"></a>
+The GUI of Tinner can be broken down into different segments with different purposes, this section would like to help users who are confused by the GUI get a better understanding of their uses and get more comfortable using the application.
```

Comment:
I think this wording is better:

The GUI of Tinner can be broken down into different segments with different purposes. This section aims to help users who are confused by the GUI get a better understanding of its use and get more comfortable using the application.